.github/workflows/pytest.yaml

```diff
@@ -91,6 +98,8 @@ jobs:
       #       check-out repo and set-up python
       #----------------------------------------------
       - uses: actions/checkout@v4
+        with:
+          fetch-depths: 0
       - uses: actions/setup-python@v5
       #----------------------------------------------
       #  -----  install & configure poetry  -----
```

Comment:
The dashes are different only for the "install & configure poetry" section headers.